gotham/src/lib.rs

```diff
@@ -15,27 +15,31 @@
 extern crate base64;
 extern crate bincode;
 extern crate borrow_bag;
+extern crate bytes;
 extern crate chrono;
 extern crate cookie;
+#[macro_use]
```

Comment:
I managed to remove the 'failure' macros - it was used in `accepted_encoding`, but the use of it could be removed too without impact it turns out.
`futures` provides the `try_ready!` macro used by the code borrowed from warp.
`serde_derive` macro is used by FilePathExtractor to extract the glob segment.

Reply:
Awesome :) sounds good to me!